test/lit/passes/remove-unused-module-elements-refs.wast

```diff
@@ -119,9 +179,13 @@
 ;; As above, but interleave CallRefs with RefFuncs.
 (module
   ;; CHECK:      (type $A (func))
+  ;; OPEN_WORLD:      (type $A (func))
   (type $A (func))
   (type $B (func))
 
+
+
+
```

Comment:
```suggestion
```

It also looks like there a number of other whitespace additions below. Are they intentional?

Reply:
Hmm, not sure what caused those... removed.

Reply:
I hope the update script doesn't put them back. Can you check?

Reply:
It doesn't, yeah. But it left them in while they were there, somehow.

Reply:
Great, that sounds like the expected behavior.